packages/app/src/systems/Bridge/components/BridgeSteps/BridgeSteps.stories.tsx

```diff
@@ -0,0 +1,56 @@
+import { cssObj } from '@fuel-ui/css';
+import { Flex, Text } from '@fuel-ui/react';
+
+import { BridgeSteps } from './BridgeSteps';
+
+export default {
+  component: BridgeSteps,
+  title: 'BridgeSteps',
+  parameters: {
+    layout: 'fullscreen',
+  },
+};
+
+export const Usage = () => {
+  const steps = [
+    {
+      name: 'Submit to bridge',
+      status: <Text fontSize="xs">Action</Text>,
+      isLoading: false,
+      isDone: false,
+      isSelected: true,
+    },
+    {
+      name: 'Settlement',
+      status: <Text fontSize="xs">Wait ~15 min</Text>,
+      isLoading: false,
+      isDone: false,
+      isSelected: false,
+    },
+    {
+      name: 'Confirm transaction',
+      status: <Text fontSize="xs">Action</Text>,
+      isLoading: false,
+      isDone: false,
+      isSelected: false,
+    },
+    {
+      name: 'Receive funds',
+      status: <Text fontSize="xs">Automatic</Text>,
+      isLoading: false,
+      isDone: false,
+      isSelected: false,
+    },
+  ];
```

Comment:
can we have one other story showing usage of this component a bit more? mixing more complex situation like:
1- done
2- selected + loading
3- normal
4- normal